aws/main.tf

```diff
@@ -44,13 +47,14 @@ module "eks" {
   # Sets additional worker security groups on console.
   cluster_create_security_group = false
   cluster_security_group_id = data.aws_security_group.cluster_sg.id
+  vpc_id       = module.vpc.vpc_id
+  enable_irsa  = true
```

Comment:
vpc_id will need to change from module.vpc_id to local.vpc_id to work with private subnets.  we could probably do that on-the-fly as part of the merge.

Reply:
Ah, OK.  Can do.